scripts/setup_ubuntu.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+sudo apt update
```

Comment:
Change this to `apt-get` instead of `apt`, since the former is backwards-compatible.